rollup.config.ts

```diff
@@ -0,0 +1,44 @@
+import replace from '@rollup/plugin-replace'
+import { defineConfig } from 'rollup'
+import rename from 'rollup-plugin-rename'
+import typescript from 'rollup-plugin-ts'
+
+export default defineConfig({
+  input: './src/index.ts',
+  plugins: [
+    replace({
```

Comment:
I understand this reassignment after thinking about it. please add a comment and combine with second replace plugin instance.

Reply:
I need the second instance since it has different delimiters, and I'm not exactly sure either if things might/could break in the future if I also changed the delimiters for the extension reassignment.

I'll go add the comments since on second look it does look a bit weird :+1:

Reply:
Oh didnt saw that. Than it is fine....